src/Annotation/README.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Annotation'
+type: 'component'
+components:
+- Annotation
+categories:
+- Status & metadata
+status: 'New'
+designStatus: 'Done'
+devStatus: 'In progress'
+notes: |
+
+---
+
+Display informative text related to an object on screen. Unlike the tooltip an annotation is not interactive.
+
+- Don't put links or interactive elements in annotations - use [Popover](/components/popover) instead
+- Don’t use annotations if trigger is needed - use [Tooltips](/components/tooltip) instead
+
+### Basic Usage
+
+```jsx live
+<>
+  <Annotation>
+    Success variant
+  </Annotation>
+  <Annotation 
+    variant="dark"
+    arrowPlacement="left"
+    className="mr-2 mb-2"
+  >
+    Dark variant
+  </Annotation>
+  <Annotation 
+    variant="error" 
+    arrowPlacement="top"
+    className="mr-2 mb-2"
+  >
+    Error variant
+  </Annotation>
+  <Annotation
+    variant="warning"
+    arrowPlacement="right" 
+    className="mb-2"
+  >
+    Warning variant
+  </Annotation>
+  <Annotation
+    variant="light"
+    className="mb-2"
+  >
+    Light variant. By default max width is set to 300px with wrapping text.
+  </Annotation>
+</>
+```
+
+### Theme variables (SCSS)
```

Comment:
With the other recently merged PR to add these theme variables automatically, they shouldn't be necessary to include in the README anymore!

Reply:
Yeah, we don't, removed it, thanks!